packages/extension-ui/src/components/FaviconBox.tsx

```diff
@@ -56,12 +56,16 @@ export default styled(FaviconBox)(
   border-radius: 8px;
   transition: 0.2s ease;
   justify-content: space-evenly;
+  
+  min-width: 0; /* To enable ellipsis for flex grandchildren (https://css-tricks.com/flexbox-truncated-text/) */
```

Comment:
I think this one isn't required

Reply:
Doesn't work without it. Also I think it makes sense for it to be required - this element has a flex parent on its own, so needs `min-width` reset as well.

Reply:
I'm sorry to bother you with such a nitpick, but I tested on chrome and firefox and it works :shrug: .
Perhaps that's a Brave thing, but would you be as kind as to check again :sweat_smile:

Reply:
Huh, you are right - I guess when checking it previously, I was mistakenly validating for `max-width` instead 🤦 Works as you're saying on both chrome and brave. Kudos for persistence 😄